tests_func/config.go

```diff
@@ -15,11 +15,12 @@ var Env = map[string]string{
 	"WALG_CONF_DIR":    "/etc/wal-g",
 	"WALG_CONF_PATH":   "/etc/wal-g/wal-g.json",
 
-	"S3_HOST":       "minio01",
-	"S3_PORT":       "9000",
-	"S3_BUCKET":     "dbaas",
-	"S3_ACCESS_KEY": "S3_ACCESS_KEY",
-	"S3_SECRET_KEY": "S3_SECRET_KEY",
+	"S3_HOST":                       "minio01",
+	"S3_PORT":                       "9000",
+	"S3_BUCKET":                     "dbaas",
+	"S3_ACCESS_KEY":                 "S3_ACCESS_KEY",
+	"S3_SECRET_KEY":                 "S3_SECRET_KEY",
+	"S3_REQUEST_ADDITIONAL_HEADERS": `{"X-Yandex-Prioritypass":"ok"}`,
```

Comment:
We support a nested configuration in WAL-G.

So you can define these headers as YAML to avoid parsing.

```
S3_REQUEST_ADDITIONAL_HEADERS:
    - X-Yandex-Prioritypass: "ok"
    - header2: val2
```

and so on...

Reply:
But from the other hand, it might be difficult to set these headers from environment, maybe the current approach is also fine.

Reply:
We thought about it with @x4m  and decided that json will be ok

Reply:
Well, at some point we can consider env vars old fashined way, and config file as a future.
Is it possible to make it interchangable?
so that both ways would work?
`S3_REQUEST_ADDITIONAL_HEADERS:
    - X-Yandex-Prioritypass: "ok"
    - header2: val2`
 and `S3_REQUEST_ADDITIONAL_HEADERS='{'X-Yandex-Prioritypass':'ok'}"`
and
`"S3_REQUEST_ADDITIONAL_HEADERS": '{"X-Yandex-Prioritypass":"ok"}'`